### PR TITLE
Alice info endpoint

### PR DESCRIPTION
**Pull request by timestruck**

### CODE REVIEW

File: bundler/intermediates/CADisigI1CertificationService_cb619f69c2d820b4bc23e4d4339f06a96d6373d4_021c8d95a0308df09c0000000000611515.crt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```
Comment (grittygrease) [inline bundler/intermediates/CADisigI1CertificationService_cb619f69c2d820b4bc23e4d4339f06a96d6373d4_021c8d95a0308df09c0000000000611515.crt]: These new certificates don't seem necessary.

**Comment by grittygrease** [comment]
Can you squash this into one commit so it doesn't conflict with the other PR about testcode?

**Comment by kisom** [comment]
Build failure: `api/generator/generator_test.go:143: github.com/cloudflare/cfssl/csr.Generator composite literal uses unkeyed fields`

File: cmd/cfssl/cfssl.go [hunk 2]
```diff
@@ -27,7 +27,10 @@ import (
 	"github.com/cloudflare/cfssl/cli/bundle"
 	"github.com/cloudflare/cfssl/cli/gencert"
 	"github.com/cloudflare/cfssl/cli/genkey"
+<<<<<<< HEAD
```
Comment (dr107) [inline cmd/cfssl/cfssl.go]: You should probably fix these merge conflicts.

File: signer/universal/universal_test.go [hunk 3]
```diff
@@ -1 +1,21 @@
 package universal
+
+import (
+	"testing"
+)
+
+func newPKCS11Signer(t *testing.T) {
```
Comment (dr107) [inline signer/universal/universal_test.go]: I'm pretty sure go test function names have to begin with "Test", then an uppercase letter to actually be run by `go test`. Try renaming this to TestNewPKCS11Signer.

**Pull request closed without merging**